packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -124,6 +126,7 @@ export function BlockSettingsDropdown( {
 				openedBlockSettingsMenu: getOpenedBlockSettingsMenu(),
 				isContentOnly:
 					getBlockEditingMode( firstBlockClientId ) === 'contentOnly',
+				isNavigationMode: _isNavigationMode(),
```

Comment:
Do we need this check? `isContentOnly` should only be true for any block when navigation mode is enabled.

Reply:
Container blocks can set editing mode outside the navigation using the `templateLock`. I guess it depends on whether to hide/show features based on only block editing mode or navigation mode.

Here's a case where both mode checks are needed - #70946.

Reply:
Hmm that feels like a bug, `templateLock` is a different thing from `editingMode` (or it should be).